sympy/functions/special/bessel.py

```diff
@@ -330,6 +330,12 @@ def eval(cls, nu, z):
                 return S.NaN
         if z in (S.Infinity, S.NegativeInfinity):
             return S.Zero
+        if abs(z) is S.Infinity:
+            if im(z) > 0:
+                coeff = I*sqrt(-I)*exp(-I*(-pi*nu/2 - pi/4))
```

Comment:
This can be simplified. `sqrt(-I)` is canceled by `exp(I*pi/4)`. Also `I = exp(I*pi/2)` so we get `exp(I*pi*(nu + 1)/2)`. There should be no need for `.n()`.

Reply:
Yeah I put the `.n()` just in case . Surely the expressions can be simplified further. I've made the changes in the latest commit .